client/src/components/profileInfoEdit/ProfileInfoEdit.js

```diff
@@ -21,22 +21,49 @@ const ProfileInfoEdit = () => {
   const dispatch = useDispatch();
 
   const {
-    teamId, teamName, orgName, email,
+    teamId, teamName, orgName, email, profilePic,
   } = useSelector(
     (state) => state.team,
   );
 
-  const [profileData, setInputs] = useState({ teamName, orgName, email });
+  const [profileData, setInputs] = useState({
+    teamName, orgName, email, profilePic,
+  });
 
   useEffect(() => {
-    setInputs({ teamName, orgName, email });
-  }, [email, orgName, teamName]);
+    setInputs({
+      teamName, orgName, email, profilePic,
+    });
+  }, [email, orgName, teamName, profilePic]);
 
   const updateInputs = (form) => {
     const { name, value } = form.target;
     setInputs({ ...profileData, [name]: value });
   };
 
+  /**
+   * Updates profile image field when user uploads file
+  */
+
+  // If profilePic is undefined, set a default profile pic
+  if (!profileData.profilePic) {
+    profileData.profilePic = defaultProfilePic;
+  }
```

Comment:
Can refactor to:
```javascript
profileData.profilePic = profileData.profilePic ?? defaultProfilePoc;
```

see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
Weird bug, doesn't display the defaultProfilePic if I use this